docs/concepts/storage.md

```diff
@@ -4,120 +4,118 @@ title: Storage
 sidebar_label: Storage
 ---
 
-## Durability and Data Recovery
+## Durability and data recovery
 
 Memgraph uses two mechanisms to ensure the durability of the stored data:
 
-  * write-ahead logging (WAL) and
-  * taking periodic snapshots.
+  * write-ahead logging (WAL)
+  * periodic snapshots
 
-Write-ahead logging works by logging all database modifications to a file.
-This ensures that all operations are done atomically and provides a trace of
-steps needed to reconstruct the database state.
+In write-ahead logging, all database modifications are recorded in a log file
+before being applied to the database. WAL ensures that all operations are done
+atomically and provides steps needed to reconstruct the database state.
 
 Snapshots are taken periodically during the entire runtime of Memgraph. When
 a snapshot is triggered, the whole data storage is written to disk. The
 snapshot file provides a quicker way to restore the database state.
 
-Database recovery is done on startup from the most recently found snapshot
-file. Since the snapshot may be older than the most recent update logged in
-the WAL file, the recovery process will apply the remaining state changes
-found in the said WAL file.
+You can also generate a snapshot file by using Memgraph's import tools. For more
```

Comment:
I'm not sure if snapshots generated by import tools fall into this category. Can you please double-check this?